cfgov/permissions_viewer/views.py

```diff
@@ -0,0 +1,91 @@
+from collections import namedtuple, OrderedDict
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.contrib.contenttypes.models import ContentType
+from django.shortcuts import get_object_or_404, render
+
+from wagtail.wagtailcore.models import Page, Collection
+
+
+User = get_user_model()
+PagePermissions = namedtuple('PagePermissions', ['page', 'group_permissions'])
+CollectionPermissions = namedtuple('CollectionPermissions',
+                                   ['collection', 'group_permissions'])
+CTPermissions = namedtuple('CTPermissions',
+                           ['content_type', 'group_permissions'])
+
+
+def display_group_roster(request, group_id):
+    group = get_object_or_404(Group, pk=group_id)
+
+    active_users = group.user_set.filter(is_active=True)
+    inactive_users = group.user_set.filter(is_active=False)
+    return render(request, 'permissions_viewer/group.html', {
+        'active_users': active_users,
```

Comment:
Would you mind adding an `.order_by('username')` on these, so that they appear in order in the admin?